### DISCUSSION TITLE
Open and edit existing picture.

### DISCUSSION THREAD

**Discussion by hornetster** [flameshot-org/flameshot#3742] [Q&A] [answered]
Is there any way to open an existing (ie on disk), picture with Flameshot?
Just want to add a couple of boxes, and a bit of text to an original pic.
Thanks.

**Comment by mmahmoudian** [comment, marked as answer]
No, such thing is not yet implemented It have been suggested in #240 and in #3676 . But until that is implemented, you can open the image with some image viewer (for example on Linux you can try [imv](https://sr.ht/~exec64/imv/) or [pqiv](https://github.com/phillipberndt/pqiv/)) and take another screenshot from it and add the annotations you want.

**Reply by hornetster** [reply]
Yeah, I want to keep all the metadata of the original picture...
Ah, well.

**Reply by mmahmoudian** [reply]
In that case perhaps the most appropriate tool is Inkscape. It is an OpenSource vector graphics tool and you can draw the boxes and arrows and text easily and quickly on existing images:

https://inkscape.org/
